src/test/jtx/impl/Env.cpp

```diff
@@ -271,7 +271,9 @@ Env::parseResult(Json::Value const& jr)
         jr[jss::result].isMember(jss::engine_result_code))
         ter = TER::fromInt(jr[jss::result][jss::engine_result_code].asInt());
     else
-        ter = temINVALID;
+        // Use an error code that is not used anywhere in the transaction engine
+        // to distinguish this case.
+        ter = telLOCAL_ERROR;
```

Comment:
"it is possible that another server with a different configuration or load level could process the transaction successfully." Is this true for any of these new situations returning `telLOCAL_ERROR`? If not, should we maybe just consider adding a new error code?

Reply:
I considered adding a new error code, but this particular result is only used in tests, so it didn't seem worth it. There are only a few test cases that expect this result (now that I've changed it), but it can also pop up when then host machine gets so busy that the standalone server doesn't answer requests. That leads to failed tests. The reason I changed it is that the previous `temINVALID` result _is_ returned by the transaction engine, which would lead on many wild goose chases as someone tried to figure out how a perfectly valid looking transaction flagged as malformed until somebody finally remembers, "Ooooh! This can also happen in `Env`."

I changed it to a `tel` because that's more accurately what it is: The communication failed locally.